pkg/ccl/testccl/sqlccl/sqlccl_test: TestTenantTempTableCleanup failed

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5981471?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5981471?buildTab=artifacts#/) on master @ [d687b14e513a49f94e1862c78a7685b78f4fbad9](https://github.com/cockroachdb/cockroach/commits/d687b14e513a49f94e1862c78a7685b78f4fbad9):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (21.76s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18362

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5981473?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5981473?buildTab=artifacts#/) on master @ [d687b14e513a49f94e1862c78a7685b78f4fbad9](https://github.com/cockroachdb/cockroach/commits/d687b14e513a49f94e1862c78a7685b78f4fbad9):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (37.43s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5992249?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5992249?buildTab=artifacts#/) on master @ [db95c2484a9f15c2de01466af0aae19c8ab57618](https://github.com/cockroachdb/cockroach/commits/db95c2484a9f15c2de01466af0aae19c8ab57618):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.30s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5992252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5992252?buildTab=artifacts#/) on master @ [db95c2484a9f15c2de01466af0aae19c8ab57618](https://github.com/cockroachdb/cockroach/commits/db95c2484a9f15c2de01466af0aae19c8ab57618):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (34.89s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5998082?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5998082?buildTab=artifacts#/) on master @ [a7c91f06d8ee0fa2096bcd626f689009024947bb](https://github.com/cockroachdb/cockroach/commits/a7c91f06d8ee0fa2096bcd626f689009024947bb):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (21.65s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5998084?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5998084?buildTab=artifacts#/) on master @ [a7c91f06d8ee0fa2096bcd626f689009024947bb](https://github.com/cockroachdb/cockroach/commits/a7c91f06d8ee0fa2096bcd626f689009024947bb):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (37.84s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6005155?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6005155?buildTab=artifacts#/) on master @ [ce8bfd4811e789bdd4cf31640722517a5cf46237](https://github.com/cockroachdb/cockroach/commits/ce8bfd4811e789bdd4cf31640722517a5cf46237):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (23.08s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6005157?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6005157?buildTab=artifacts#/) on master @ [ce8bfd4811e789bdd4cf31640722517a5cf46237](https://github.com/cockroachdb/cockroach/commits/ce8bfd4811e789bdd4cf31640722517a5cf46237):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (36.51s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6020322?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6020322?buildTab=artifacts#/) on master @ [283035ce93c023431846a0b6d4e2f38a5bb88142](https://github.com/cockroachdb/cockroach/commits/283035ce93c023431846a0b6d4e2f38a5bb88142):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (21.85s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6020324?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6020324?buildTab=artifacts#/) on master @ [283035ce93c023431846a0b6d4e2f38a5bb88142](https://github.com/cockroachdb/cockroach/commits/283035ce93c023431846a0b6d4e2f38a5bb88142):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (37.74s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6035716?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6035716?buildTab=artifacts#/) on master @ [6c24f3564e213371bbe800e9646386cdefeb1546](https://github.com/cockroachdb/cockroach/commits/6c24f3564e213371bbe800e9646386cdefeb1546):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6035718?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6035718?buildTab=artifacts#/) on master @ [6c24f3564e213371bbe800e9646386cdefeb1546](https://github.com/cockroachdb/cockroach/commits/6c24f3564e213371bbe800e9646386cdefeb1546):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (36.19s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6048628?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6048628?buildTab=artifacts#/) on master @ [d266989c4d30e6fbdc3fd6ac7458e7a651b87131](https://github.com/cockroachdb/cockroach/commits/d266989c4d30e6fbdc3fd6ac7458e7a651b87131):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (34.95s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6048626?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6048626?buildTab=artifacts#/) on master @ [d266989c4d30e6fbdc3fd6ac7458e7a651b87131](https://github.com/cockroachdb/cockroach/commits/d266989c4d30e6fbdc3fd6ac7458e7a651b87131):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (24.26s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6065336?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6065336?buildTab=artifacts#/) on master @ [773f7d4445ce3e0e806b7a182adba70a0f270f19](https://github.com/cockroachdb/cockroach/commits/773f7d4445ce3e0e806b7a182adba70a0f270f19):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (37.75s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6065334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6065334?buildTab=artifacts#/) on master @ [773f7d4445ce3e0e806b7a182adba70a0f270f19](https://github.com/cockroachdb/cockroach/commits/773f7d4445ce3e0e806b7a182adba70a0f270f19):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.64s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6079959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6079959?buildTab=artifacts#/) on master @ [0dd438d3dc0b42543890455945a7a6b42811def1](https://github.com/cockroachdb/cockroach/commits/0dd438d3dc0b42543890455945a7a6b42811def1):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (21.39s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6079961?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6079961?buildTab=artifacts#/) on master @ [0dd438d3dc0b42543890455945a7a6b42811def1](https://github.com/cockroachdb/cockroach/commits/0dd438d3dc0b42543890455945a7a6b42811def1):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (36.92s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6087812?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6087812?buildTab=artifacts#/) on master @ [d25cb57ccd9bc643ce9058ebd2057cab36b69ad5](https://github.com/cockroachdb/cockroach/commits/d25cb57ccd9bc643ce9058ebd2057cab36b69ad5):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.37s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6087815?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6087815?buildTab=artifacts#/) on master @ [d25cb57ccd9bc643ce9058ebd2057cab36b69ad5](https://github.com/cockroachdb/cockroach/commits/d25cb57ccd9bc643ce9058ebd2057cab36b69ad5):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (38.28s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6094970?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6094970?buildTab=artifacts#/) on master @ [6b7f0058325de8b8b321a08137d462c418c3047f](https://github.com/cockroachdb/cockroach/commits/6b7f0058325de8b8b321a08137d462c418c3047f):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6094972?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6094972?buildTab=artifacts#/) on master @ [6b7f0058325de8b8b321a08137d462c418c3047f](https://github.com/cockroachdb/cockroach/commits/6b7f0058325de8b8b321a08137d462c418c3047f):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (38.49s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108484?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108484?buildTab=artifacts#/) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (20.97s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108486?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108486?buildTab=artifacts#/) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (38.58s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124534?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124534?buildTab=artifacts#/) on master @ [b173a16715e71e94115820374da1eb350b3b459d](https://github.com/cockroachdb/cockroach/commits/b173a16715e71e94115820374da1eb350b3b459d):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (35.72s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestTenantTempTableCleanup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124533?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124533?buildTab=artifacts#/) on master @ [b173a16715e71e94115820374da1eb350b3b459d](https://github.com/cockroachdb/cockroach/commits/b173a16715e71e94115820374da1eb350b3b459d):


```
=== RUN   TestTenantTempTableCleanup
    temp_table_clean_test.go:145: query 'SELECT table_name FROM [SHOW TABLES]': expected:
        temp_table
        
        got:
        temp_table
        temp_table2
        
--- FAIL: TestTenantTempTableCleanup (22.30s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantTempTableCleanup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/testccl/sqlccl/temp_table_clean_test.go
+++ b/pkg/ccl/testccl/sqlccl/temp_table_clean_test.go
@@ -142,7 +142,7 @@ func TestTenantTempTableCleanup(t *testing.T) {
 	// two clean up cycles just in case, so that we have
 	// stable timing.
 	waitForCleanup()
-	tenantSQL.CheckQueryResults(t, "SELECT table_name FROM [SHOW TABLES]",
+	tenantSQL.CheckQueryResultsRetry(t, "SELECT table_name FROM [SHOW TABLES]",
 		[][]string{
 			{"temp_table"},
 		})

